### PR TITLE
Math-Magicians: components

### PR DESCRIPTION
**Pull request by Melaku05**
-create  UI for the calculator
.create components

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,16 +1,17 @@
 {
-  "name": "my",
+  "name": "do-the-math",
```
Comment (KDlamini) [inline package-lock.json]: On startup, your application fails to run when I run `npm run start` I get errors. It worked after I deleted the `package-lock.json` file and the `node_modules` and ran `npm install` again.

- Possible reasons could be that you pushed the `node_modules` directory into your GitHub repo. I see that you included `node_modules` in your `.gitignore` file. But somehow it got pushed to your GitHub repo. Kindly remove it from your GitHub and make sure that it doesn't get pushed again.
image

- When a user clones your repository, they should not have the `node_modules` directory until they install it locally using `npm install`.
- After debugging the issues, kindly include any other steps that need to be taken in the README to make your application run if any.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 # MATH-MAGICIANS
```
Comment (KDlamini) [inline README.md:4]: -  Kindly add a description for your project.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Melaku05 🙋‍♀️,  

Great job so far! 
You have done a great job implementing most of the project requirements. 🎊 🥳 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Correct PR title
✔️  Correct use of Gitflow 
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Melaku05) [reply README.md:4]: Thank you, I did that.

File: package-lock.json (hunk 1 above)
Comment (Melaku05) [reply package-lock.json]: Thank you, I did that.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Melaku05, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**